### DISCUSSION TITLE
Will the model remember the relevant and irrelevant paper after reboot?

### DISCUSSION THREAD

**Discussion by PengKuang** [asreview/asreview#750] [Q&A]
Hi there, thanks for developing such an interesting and useful tool. 

I played with the tool a little bit using a dataset I intend to use for a review study. And I used a small sample dataset to train it by having created a toy project.

Now I want to get serious about using it. I want to further divide my sample dataset into a training dataset and a test dataset so that I can manually examine how accurate the predictions could be. But since I have already used some of the papers (actually most of the relevant papers) in the toy project, I wonder if the tool will still treat them all as novel if I recreate a project, re-upload the dataset, and re-label the relevant and irrelevant papers (ofc this time just less in total) as the prior knowledge?  I hope it will be so:)

Thanks in advance for your reply!

**Comment by ottomattas** [comment]
Hi @PengKuang. Thanks for taking up this discussion and looking into using the tool more extensively!

> Now I want to get serious about using it. I want to further divide my sample dataset into a training dataset and a test dataset so that I can manually examine how accurate the predictions could be. But since I have already used some of the papers (actually most of the relevant papers) in the toy project, I wonder if the tool will still treat them all as novel if I recreate a project, re-upload the dataset, and re-label the relevant and irrelevant papers (ofc this time just less in total) as the prior knowledge? I hope it will be so:)

If I now understood you completely right, you will be creating a new project. Each project setup creates a new "ASReview file". This will be stored on your local disk, separately from your other projects. Even if the input dataset is identical to the one you used for another project, there will be no overlap. This means that the model will be created from scratch each time.

Note. Please, do not confuse this workflow with exporting/importing a project.

Let us know if I answered your question? Any new question is also welcome  :)

**Reply by PengKuang** [reply]
Hi Otto,

Thanks for your reply. I followed your instruction to create my new project but I encountered some strange issues:
Context: I used 100 papers as the dataset for my previous toy project. I didn't delete this toy project. For the new project, I used 50 of these 100 papers as the dataset.

1. I couldn't find some of the 50 papers in my new project. The system doesn't return the correct paper to me when I search a paper by its title. However, when I was doing this in the toy project, it always hit the correct paper.
2. when I started screening in the new project, the system gave me papers that are out of the 50 papers which I provided as the dataset. Some of them are from the rest of the 100 papers in my toy project. Some of them are not even in these 100 papers. How could this happen? Is the system actually searching on the web for relevant papers?

**Reply by PengKuang** [reply]
just wanted to correct myself:
1. "Some of them are not even in these 100 papers"
After double-checking, it turns out these papers are from the 100 paper dataset as well.

**Reply by ottomattas** [reply]
Hi @PengKuang,
Thanks for running some more tests. Is it possible for you to share your datasets with us so we can try reproducing the issue?

(I would like to confirm the issue by taking the exact actions you took. And then debug the issue to understand what might be going wrong.)

**Reply by ottomattas** [reply]
Hi @PengKuang, did you succeed with your actions?
If yes, maybe you can hint at what was the problem so the community can learn, too?
